new/PRS14031John.xml

```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14031John" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>John of Parallos</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-05-30">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1">John of Parallos</persName>
+                    <persName xml:lang="gez" xml:id="n2"><roleName type="title">አባ፡ </roleName>ዮሐንስ፡ </persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2"><roleName type="title">Abbā</roleName> Yoḥannəs</persName>
+                    <persName xml:lang="gez" xml:id="n3"><roleName type="title">አባ፡ </roleName>ዮሐንስ፡ ዘሀገረ፡ ቡርልስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n3"><roleName type="title">Abbā</roleName> Yoḥannəs zahagara Burləs</persName>
+                    <faith type="Coptic"/>
+                    <occupation type="ecclesiastic">Bishop of Nikiu</occupation>
+                    <nationality type="Egypt"/>
+                    <birth>ca. 540</birth>
```

Comment:
`birth when="0540" `always try to encode dates
same for death

Reply:
How can I record the "ca.". If I remember right the file, that you send me or another source gave "c. 540". I interpreted this as "ca. 540", what means, that this is an approximate date. It may be a couple of years before or later. May I encode it as cert="medium" or atLeast="0535" atMost="0545". What is correct or what is better?

Reply:
use `cert` and leave the text `<birth when="0540" cert="low">ca. 540</birth>`